clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -78,7 +83,8 @@ trait GvcfTests { self: BaseIntegrationSpec =>
       sampleAlias = s"someAlias$randomId",
       version = 2,
       documentStatus = Some(DocumentStatus.Normal),
-      gvcfPath = Some(URI.create("gs://path/gvcf.gvcf"))
+      gvcfPath = Some(URI.create("gs://path/gvcf.gvcf")),
+      regulatoryDesignation = Some(RegulatoryDesignation.ResearchOnly)
```

Comment:
Could this be something other than `ResearchOnly`, to show that the server doesn't overwrite the value if it's given?

Reply:
This is the expected value we want to get back from the server

Reply:
This is the expected GVCF that the GVCF returned from the server is being tested against.

Reply:
Ah, missed that this was in the `expected`. Could we still change this to something else, and then pass the same something else in with the metadata? This sort of test would've caught the other problem I commented on.

Reply:
like this? https://github.com/broadinstitute/clio/pull/164/files/802a5c9dd95f5a1f5973736c1ca7d7f2f04de913#diff-a369a91bf3d67fb7ffd26a123a61aa14R795

Reply:
No, as in a test where you set it to something other than RESEARCH_ONLY and it isn't changed by the server.

Reply:
ah ok.